hybris/custom/klarnapayment/src/com/klarna/payment/converter/populator/KPCreditSessionPopulator.java

```diff
@@ -51,7 +54,7 @@ public void populate(final AbstractOrderModel source, final PaymentsSession targ
 		LogHelper.debugLog(LOG, "inside full populator");
```

Comment:
Debug log should not be in production code

Reply:
Debug logs have been removed
Commit can be found in https://github.com/klarna/SAP-Commerce-klarna-payments/commit/c3d029e9830e600544e03277455cf385023f4a5c